backend/Helpers/Hasher.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+
+namespace OpenData.Helpers
+{
+    public static class Hasher
```

Comment:
This should not be a static class, you should use dependency injection for this.

ISecurityProvider, and then SecurityProvider.